frontend/src/components/maps/leaflet/InfoSlideOut/AssociatedTab.test.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import 'leaflet';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme from 'enzyme';
+import 'leaflet/dist/leaflet.css';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router-dom';
+import thunk from 'redux-thunk';
+import configureMockStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import { IParcel, IBuilding } from 'actions/parcelsActions';
+import { render } from '@testing-library/react';
+import { mockParcel, mockBuilding } from './InfoContent.test';
+import AssociatedParcelsList from './AssociatedParcelsList';
+import { Label } from 'components/common/Label';
+import { FaPlusSquare } from 'react-icons/fa';
+import AssociatedBuildingsList from './AssociatedBuildingsList';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
This probably just got carried over from an older test, but I don't think Enzyme is actually being used here!

Reply:
right you are! lol